test/es-module/test-importmap.mjs

```diff
@@ -0,0 +1,203 @@
+// Flags: --expose-internals
+
+import { spawnPromisified } from '../common/index.mjs';
+import fixtures from '../common/fixtures.js';
+import tmpdir from '../common/tmpdir.js';
+import { describe, it } from 'node:test';
+import assert from 'node:assert';
+import path from 'node:path';
+import { execPath } from 'node:process';
+import { pathToFileURL } from 'node:url';
+import { writeFile } from 'node:fs/promises';
+import http from 'node:http';
+import import_map from 'internal/modules/esm/import_map';
+const { ImportMap } = import_map;
+import binding from 'internal/test/binding';
+const { primordials: { SafeMap, JSONStringify } } = binding;
+
+const importMapFixtureRoot = fixtures.path('es-module-loaders', 'importmaps');
+const entryPoint = pathToFileURL(path.resolve(importMapFixtureRoot, 'index.mjs'));
```

Comment:
The `fixtures.path` and `fixtures.fileURL` methods let you avoid doing lots of path manipulation.

```suggestion
const entryPoint = fixtures.path('es-module-loaders/import-maps/index.mjs');
```

Reply:
Yep I did not know about the `fixtureURL one. I will look at moving to those.